app/scripts/main.min.js

```diff
@@ -165,12 +165,19 @@ const Installer = function(root) {
   root.addEventListener('touchend', install.bind(this));
 };
 
-const AirHorn = function(root) {
+const AirHorn = function(root, isInstalled) {
```

Comment:
Why not just have this calculated inline in this function rather than passed in?

Reply:
Fair. I did something else before, and the passed in variable is an artifact. https://github.com/GoogleChromeLabs/airhorn/pull/50/commits/3bef1cf89406a27917a6ab224f661404c3d0f72c now inlines it.